src/main/java/Duke.java

```diff
@@ -7,6 +7,7 @@ public static void main(String[] args) {
         int count = 0;
         int count2 = 0;
         int count3 = 1;
+        int count4 = 0;
         String content2;
         String content3;
         boolean isProblem = false;
```

Comment:
Instead of declaring a isProblem variable, you can put the try & catch in the while loop, and then the else statement in the while loop (that executes when there is no problem) into the try & catch.